pages/doc-components/CodeContainer.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import {LiveCode as MDXLive} from 'mdx-live'
+import copy from 'clipboard-copy'
+
+import {Relative, Absolute, Link, StyledOcticon} from '../..'
+import {Clippy, Check} from '@githubprimer/octicons-react'
+
+function CodeContainer({children: code, className: language}) {
+  if (language === 'language-.jsx') return <LiveCode initialCode={code} />
+  else return <StaticCode code={code} />
```

Comment:
I don't think the `else` is necessary here because the if statement above has a `return`.
```suggestion
  return <StaticCode code={code} />
```

Reply:
THIS IS NOT ME

Reply:
THIS IS AN IMPOSTER WITH MY PICTURE

Reply:
COLE????? WHAT BUTTON WHAT #463